docs/tools/chrome-extension/pr-description.md

```diff
@@ -1,39 +1,39 @@
 ---
 id: pr-description
 title: 'PR Description'
-sidebar_label: 'PR Description'
+sidebar_label: PR Description
 keywords:
   - ai
   - pr descriptions
 ---
 
-This will help you to create a PR based on the configuration you have in the extension, it will enable you to create the summary based on either **Commit Messages** or **The diff between the files** or **both**, you can even set the length and the tone of the message using the same settings.
+The OpenSauced Chrome extension will help you create a PR based on the configuration you have in the extension. It will enable you to create the summary based on either commit messages, the difference between the files, or both. You can even set the message's length and tone using the same settings.
 
-_Note: This feature is not available on private repositories._
+_**Note**: This feature is not available on private repositories._
 
 ![generate pr description](../../../static/gif/pr-description.gif)
 
 You can use this settings to set the strictness and the tone of the generated AI, here is an explanation of those settings:
 
-- **Description** length:The output length.
-- **Temperature**: Is the similarity between the input text and the output, higher temperature mean more randomness, lower temperature means more strict to the input.
-- **Max Input length.**
-- **Description language:** right now this feature supports: English, Spanish, French, German, Italian, Portuguese, Dutch, Russian, Chinese, Korean.
-- **Description tone:** The tone of the output: Exciting, Persuasive, Informative, Humorous, Formal
-- **Description source:** The source of the description that our AI will use to get the output it can be the next:**Commit Messages** or **The diff between the files** or **both**
+- **Description Length**: The output length.
+- **Temperature**: The similarity between the input text and the output, higher temperature mean more randomness, lower temperature means more strict to the input.
+- **Max Input Length.**
+- **Description Language:** Right now, this feature supports: English, Spanish, French, German, Italian, Portuguese, Dutch, Russian, Chinese, Korean.
+- **Description Tone:** The tone of the output: Exciting, Persuasive, Informative, Humorous, Formal
+- **Description Source:** The source of the description that our AI will use to get the output it can be the next: **Commit Messages** or **The diff between the files** or **both**
 
 ![Setting screenshot](https://user-images.githubusercontent.com/18273833/241760972-aa10eabe-3c01-4921-956a-ab85bada1575.png)
 
 ![ezgif-3-43e056fc1f](https://github.com/open-sauced/docs.opensauced.pizza/assets/18273833/fa19fa8e-e652-461c-8df4-6e959c9b9943)
 
 [Example PR](https://github.com/open-sauced/insights/pull/1197), using the next options: both in the description source, 500 for output length and 3900 max input length, temperature 0.7.
```

Comment:
I'm debating to give an example here, and leaning towards removing this part entirely. The reasons are:

- When we click the PR, we just see a PR. I mean, we can't see the extension in action.
- Even though it says here that this PR is using both commit messages and diff, I doubt user would check the commit messages and file differences one by one. 

I think the first GIF gives enough explanation on how the extension works in crating PR description.